### DISCUSSION TITLE
how to use it in arduino ide ?

### DISCUSSION THREAD

**Discussion by tripod2** [UncleRus/esp-idf-lib#451] [Q&A] [answered]
this seem better to use a library for all the sensor i experimented lot of icompatibility when trying to setup a screen with sensors
this list is impressive but i need some exemple because im a beginner 
where can i found it ?

**Comment by trombik** [comment, marked as answer]
the libraries in the repository are all C libraries. Arduino is (a variant of) C++. a C++ application can use C libraries. include one in your arduino code.

```c++
#include <whatever.h>
```

replace `whatever.h` with the one you want to use. in theory, all the example should work if you paste `main()` in the example into `setup()`, and other functions to outside of `setup()`.

no, we don't have examples for Arduino.

**Comment by tripod2** [comment]
you mean none ever used this library for arduino ?
its sad to have so much data unused
